developerportal/apps/content/templates/content.html

```diff
@@ -6,18 +6,32 @@
 {% block body_class %}content{% endblock %}
 
 {% block content %}
-  <main>
+  <main class="{% if page.hero_image %}content-has-hero{% endif %}">
```

Comment:
We should be able to use the `has-header` class for this.

Reply:
Ok done.